db/constants.go

```diff
@@ -2,4 +2,5 @@ package db
 
 const dbEndpoint string = "http://localhost:8000/"
 const dbTableNameItems string = "items"
-const dbTableNameLists string = "lists"
+
+// const dbTableNameLists string = "lists"
```

Comment:
This intentional?

Reply:
If we don't need it can we just delete it?

Reply:
The linting didn't like it because it's unused, I didn't bother doing much with it because it's removed in this PR: https://github.com/mount-joy/thelist-lambda/pull/7